feature.html

```diff
@@ -0,0 +1,115 @@
+<html>
+<link href="css/feature.css" rel="stylesheet">
+<div class="feature">
+    <div class="sectionheading">
+        <div class="sectionheading1">
+            <div class="sectiontitle">
+                <text>Tailor-made features</text>
+            </div>
+        </div>
+        <div class="description">
+            <p>Lorem ipsum is common placeholder text used to demonstrate the graphic elements of a document or
+                visual presentation.</p>
+        </div>
+
+    </div>
+    <div class="feturelist">
+        <div class="row1">
+            <div class="section1">
+                <div class="icon1">
+                    <img src="images/01.jpg">
+                </div>
+                <div class="details1">
+                    <div class="sectionheadin1">
+                        <text1>Robust workflow</text1>
+                        <div class="description1">
+                            <p1>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed erat nibh tristique ipsum.
+                            </p1>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="section2">
+                <div class="icon2">
+                    <img src="images/02.jpg">
+                </div>
+                <div class="detail2">
+                    <div class="sectionheadin2">
+                        <text2>Flexibility</text2>
+                        <div class="description2">
+                            <p2>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed erat nibh tristique ipsum.
+                            </p2>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="section3">
+                <div class="icon3">
+                    <img src="images/03.jpg">
+                </div>
+                <div class="detail3">
+                    <div class="sectionheadin3">
+                        <text3>User friendly</text3>
+                        <div class="description3">
+                            <p3>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed erat nibh tristique
+                                ipsum.
+                            </p3>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="row2">
+            <div class="section4">
+                <div class="icon4">
+                    <img src="images/04.jpg">
```

Comment:
give meaningful name to all images
update all images accordingly

Reply:
i have multiple images with same logo. so to clear my confusion i gave numbers. i will makesure from nexttime.